lab-matt/cowsay.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
This is a good start but lacks essential features specified by the assignment.  Please see my comments below, additionally this file should be renamed to server.js, cowsay.js is an ambiguous file name.